NOnion/TorStream.fs

```diff
@@ -1,12 +1,10 @@
 ﻿namespace NOnion
 
-open FSharpx.Control.Observable
+open System
+open System.Collections.Concurrent
 
 open NOnion.Cells
 open NOnion.Utility
-open FSharp.Control.Reactive
```

Comment:
this open statement is being added in the 1st commit, but it should be added only on the commit that adds system.reactive support

Reply:
fixed